### ISSUE TITLE
hidden augm. dots behind other shapes

### ISSUE THREAD

**Issue by Bacchushlg** [Audiveris/audiveris#455]
In rather seldom cases it happens that augm. dots are hidden behind other shapes, preferably behind an end of a slur.
Sometimes I search for a couple of time for the reason that the rhythm check makes a measure staying in rose colour...

The easiest solution to make the recognition easier would be to blow up the size of augm. dots so that they are always visible "behind" other shapes (just twice the size seems to be a good size. Or maybe some sort of shadow around (in 50% transparency, also double size).

**Comment by Bacchushlg** [comment]
Unfortunately I don't find an example for this problem any more. (Damn that I did not add it when creating the issue).
The idea is just to have a bigger representation of an augmentation dot so that it is visible even when it is "behind" the beginning of a slur, which is sometimes the case (but the dot was detected wrong, because it was just the beginning a the slur). This only applies to bad image sources.

**Comment by hbitteur** [comment]
Let's postpone this until you find concrete examples.
Satisfied with that deal? :-)

**Comment by Bacchushlg** [comment]
I'm sure that I had this problem (not only once).
My idea was just to increase the representation of the aug. dots. I think that a diameter twice of the line width of slurs would be a good idea - this should assure that the dots are visible - even if they are directly collated to a slur.

**Comment by hbitteur** [comment]
Finally implemented in commit ad5b454ddc83fd39b26f3d75df71e8640ea2fef4 on development branch.

Standard display:

![image](https://github.com/Audiveris/audiveris/assets/22053634/1c51fc48-8c67-4643-8dbc-764ec1051a8b)

The end-user can toggle the "jumbo" mode (using the pull-down menu`View | Show jumbo inters` or the `F7` key):

![image](https://github.com/Audiveris/audiveris/assets/22053634/e6c06dfe-1fbe-4f2a-a385-be6848fb5c1a)

Jumbo display:

![image](https://github.com/Audiveris/audiveris/assets/22053634/6a737ec1-a6ae-4ccd-acab-5d96593a7476)

Note the augmentation dots are displayed in red color using a larger font.

By default, just the augmentation dots are concerned and their display size is doubled.  
The end-user can modify this, using the `Tools | Options` dialog, then searching for "jumbo", to get access to two constants:
- `jumboClasses`: a comma-separated list of inter class names. Default is "AugmentationDotInter"
- `zoomForJumbos`: default is 2.0

**Comment by Bacchushlg** [comment]
Fine and very good visible now.
To be honest: if the colour would remain green, I would even be happier, because then I would use the jumbo mode as default.

**Comment by hbitteur** [comment]
> Another candidate would be a dot accent.

You mean a staccato dot? 
If so, I have a problem because there is no dedicated class for this shape.
There is an `ArticulationInter `class, for which the precise shape can be: TENUTO, ACCENT, STACCATO, STACCATISSIMO, MARCATO

So, currently, we could display or not all ArticulationInter instances in jumbo mode, but not just the STACCATO shape.

If "absolutely necessary", I could change the jumbo implementation to check not only the class but also the shape within the class.  
But, do we really need this? This jumbo mode is meant to temporarily make certain items very visible.

Also, I don't really understand your suggestion to keep the jumbo-displayed item with their original color. It is not consistent with the objective of being "highly visible".

Right now, you can repeatedly press the `F7` key to show/hide the jumbo items in a flashy way. The jumbo mode is not meant to be permanent.

**Comment by Bacchushlg** [comment]
ok, let's forget about staccato etc. Was just an idea when I read about the existing list...

About the coloured jumbo: my initial idea was to have a general augmented aug.dot. Of course your solution (F7 to toggle) is much better. But I could imagine that I would use the new jumbo mode as default during transcription. In this case the changed colour would bother me. 
Just give it a try and keep the jumbo dot in the original colour...

**Comment by hbitteur** [comment]
OK, I just pushed commit 1e5c9858d934106e27e5b099077b5bbe24be6ac8 on development branch.

There is a new constant named `jumboColored` ("_Should the jumbo items be colored specifically?_")
It set to true (the default), any jumbo item is displayed in red.
You can set it to false (using the `Tools | Options` dialog) and any jumbo will be displayed in its normal color.

**Comment by Bacchushlg** [comment]
I just tried it - and I like it very much this way. Jumbo mode will be my default for sure ;-)

**Comment by hbitteur** [comment]
@Bacchushlg 
"Tes souhaits sont des ordres"... :-)
I just pushed commit 96b76b99cc54e199efc03d4d901e83efc1de543c

It extends jumbo specifications to include, beside the Inter class name, an  optional shape name introduced by a slash,
The default specifications are now:

![image](https://github.com/Audiveris/audiveris/assets/22053634/dd14eb2c-2174-4b6a-ac80-f5017f712335)

In the example below, the detected augmentation dots and the (false) staccato dots appear very clearly:

![image](https://github.com/Audiveris/audiveris/assets/22053634/9cd6f78c-4d3b-467c-b4e7-17110b14ff37)
